src/couch/rebar.config.script

```diff
@@ -92,15 +92,6 @@ MD5Config = case lists:keyfind(erlang_md5, 1, CouchConfig) of
         []
 end,
 
-AegisConfig = case lists:keyfind(crypto_module, 1, CouchConfig) of
```

Comment:
why did this need to move?

Reply:
Macro set in confg `src/couch/rebar.config.script` defined only for modules in `couch` and aegis can't compile when it set here, failing with `macro ?AEGIS_KEY_MANAGER undefined`.